src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/LabelExtensions.cs

```diff
@@ -4,7 +4,28 @@ namespace Xamarin.Platform
 {
 	public static class LabelExtensions
 	{
-		public static void UpdateText(this TextView textView, ILabel label) =>
+		public static void UpdateText(this TextView textView, ILabel label)
+		{
 			textView.Text = label.Text;
+		}
+
+		public static void UpdateTextColor(this TextView textView, ILabel label)
+		{
+			textView.UpdateTextColor(label, null);
```

Comment:
I'm curious about these places here where we pass in null because technically null is an actual setting.

It feels like when we do calls like this it should pass in whatever value we typically use for the default

Reply:
We got that DefaultTextColor property in LabelHandler...

Reply:
For example on Switch I do this

https://github.com/xamarin/Xamarin.Forms/pull/12196/files#diff-abebb275026c8510de8beeb9b1b1cb717f886959f66de018ec1f3d5a8c772099R39

Because if no default is passed in and the color used is Color.Default

It needs to do something useful...  Setting the color to null is wrong

It should either just set it back to the default or if it's Color.Default it should maybe do nothing?

Reply:
Why does this have an option to pass in a default if we have the default on the handler?

Reply:
`public static void UpdateTextColor(this TextView textView, ILabel label)` is never called, gonna just drop it and remove the nullability in the other method.

Reply:
Yea that probably works for now

I think the idea here was that these extension methods are useful outside the concept of the Handlers so it's useful to think of these extension methods as something users will just use themselves from their own code

So having extensions that don't take default could be useful in those scenarios?

But we can probably just remove them for now